### ISSUE TITLE
protobuf v2 compatibility

### ISSUE THREAD

**Issue by bumberboy** [connectrpc/connect-query-es#398]
Hello team.

Thanks for creating and maintaining this library.

As https://github.com/bufbuild/protobuf-es moves to v2, is there a timeline for the migration of of connect-query-es to v2 as well?

**Comment by timostamm** [comment]
Yes, we want to release [connect-es](https://github.com/connectrpc/connect-es), connect-query-es, and [connect-playwright-es](https://github.com/connectrpc/connect-playwright-es) with support for [protobuf-es](https://github.com/bufbuild/protobuf-es) v2, and update the [documentation](https://github.com/connectrpc/connectrpc.com) where necessary. We plan to release it at the same time to make it easier to upgrade.

You'll be able to upgrade with the command `npx @connectrpc/connect-migrate`.

We're currently held back by fixing a couple of issues in connect-es, but hopefully it shouldn't take too long.

**Comment by kyleosborn** [comment]
So I just spent the last couple of days updating to V2 for a new project, I didn't realise this only supported V2 (oops!)

Appreciate you're working on this in the background, but do you have any rough timescales as to when we can expect this to be available?

**Comment by timostamm** [comment]
Release candidates are available: https://github.com/connectrpc/connect-query-es/releases

Closing this in advance.
